src/wasm-builder.h

```diff
@@ -434,6 +434,21 @@ class Builder {
     ret->finalize();
     return ret;
   }
+  SIMDLoadStoreLane* makeSIMDLoadStoreLane(SIMDLoadStoreLaneOp op,
+                                           Address offset,
+                                           Address align,
+                                           uint8_t index,
+                                           Expression* ptr,
+                                           Expression* vec) {
+    auto* ret = wasm.allocator.alloc<SIMDLoadStoreLane>();
+    ret->op = op;
+    ret->offset = offset;
+    ret->align = align;
+    ret->index = index;
+    ret->ptr = ptr;
+    ret->finalize();
```

Comment:
It looks like this does not assign `vec`?

(I wonder if an automatic code generation tool could help in the future here... :thinking:)

Reply:
Oops! Yes, it certainly would.